var/spack/repos/builtin/packages/py-scikit-learn/package.py

```diff
@@ -35,8 +35,9 @@ class PyScikitLearn(PythonPackage):
     version('0.15.2', 'd9822ad0238e17b382a3c756ea94fe0d')
     version('0.16.1', '363ddda501e3b6b61726aa40b8dbdb7e')
     version('0.17.1', 'a2f8b877e6d99b1ed737144f5a478dfc')
+    version('0.13.1', 'acba398e1d46274b8470f40d0926e6a4')
 
     depends_on('python@2.6:2.8,3.3:')
-    depends_on('py-setuptools',   type='build')
+    depends_on('py-setuptools',   type='build', when='@0.17.1:')
```

Comment:
It's the first version included in the recipe that needs `py-setuptools`, so no, it would most likely not be the first version that adds it. It was just a way to get around it